src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -475,8 +476,8 @@ public async Task<IActionResult> Edit(WorkflowTypeUpdateModel model)
             foreach (var activityState in activities)
             {
                 var activity = currentActivities[activityState["id"].ToString()];
-                activity.X = (int)Convert.ToDecimal(activityState["x"].ToString());
-                activity.Y = (int)Convert.ToDecimal(activityState["y"].ToString());
+                activity.X = (int)Convert.ToDecimal(activityState["x"].ToString(), CultureInfo.InvariantCulture);
```

Comment:
I'm not sure why we need a decimal while at the end we convert the values to integers.

/cc @hyzx86

Reply:
I'm not sure, maybe the X, Y attributes are defined as Integer so they need a layer of conversion, I wouldn't mind changing them directly to `Decimal`.

Reply:
I think it's better to keep X, Y attributes as int. Most browsers truncate the fractional part if the position is set with decimal numbers (10.75px -> 10px).

I've made a version where the state is returned from the client without the fractional part. It uses "Math.trunc" that is only available from javascript version es6/es2015.